packages/design-system/modal/modal.container.web.tsx

```diff
@@ -1,6 +1,8 @@
 import { memo, useMemo } from "react";
 import { StyleSheet } from "react-native";
 
+import { useEscapeKeydown } from "@radix-ui/react-use-escape-keydown";
+import FocusTrap from "focus-trap-react";
```

Comment:
this component seems to only use on the web, could you move it to `app/lib/focus-trap-react/index.web.ts`? we need to unify.